plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/LanguageServerFileTypeRegister.java

```diff
@@ -38,7 +37,6 @@
 /** @author Evgen Vidolob */
 @Singleton
 public class LanguageServerFileTypeRegister {
-  private static Logger LOGGER = Logger.getLogger(LanguageServerFileTypeRegister.class.getName());
```

Comment:
I'm curious how removing that logger is related to the improvement of the workspace status panel?

Reply:
Too much outputs in browser console when restarting workspace.

Reply:
I agree on the idea but I think there is less radical way to clean browser console for instance we can change logging level to `debug`, what do you think?

In general I share the concerns of @azatsarynnyy because changes that you have introduced here and in some other places seems to be not related to the issue that you're trying to solve and may lead to the confusion during code review. If you're still interested in it I guess it would be a good idea to move it to a separate pull request named "Too much outputs in browser console when restarting workspace"

Reply:
What's the problem to remove three lines of code which do nothing except print unnecessary information in the console? Be polite, don't commit code like this.
I have a feeling it's the biggest problem you are solving.

Reply:
I agree with @dkuleshov @azatsarynnyy these changes not related to your task and your can set DEBUG level for sure

Reply:
We use a separate commit for such purpose in the PR. But this usually works when a PR is designed to be reviewed commit by commit.

Reply:
To clarify +1 to @vitaliy-guliy  that it is not so crucial to not to clean the code up.

Reply:
IMHO If cleaning up changes are small there is no necessity to create new PR for that and it's good enough to have them just in a separated commit.
Another topic is removing such logs vs writing them with debug level. I think, debug logging should be used carefully. And it'd better not to write things that cannot help while debugging. It is just my thought, I don't know can these changes be helpful or not but for me, it's quite clear that these logs should not be logged with info level.

Reply:
@vitaliy-guliy it's not about changing the logging level or removing it at all. But it's more about adherence to the *Single Responsibility Principle* in PR. So the PR should be focused on addressing only a single concern but not all problems that have been noticed in the code during development.
Just treat it as a recommendation/best practice. Don't take it to heart :wink: